sections/monte_carlo/quantum_monte_carlo/quantum_monte_carlo.tex

```diff
@@ -69,24 +61,246 @@ \subsubsection{Amplitude Estimation}\label{sec:MC_AE}
+Furthermore $\ket{\phi_0}$ and $\ket{\phi_1}$ can be viewed as the horizontal and vertical axes of a two-dimensional Hilbert space. The angle between $\ket{U}$ and the horizontal axis is thus
+$\theta_a = \arcsin(\sqrt{a})$. Obviously we would like to rotate the initial state $\ket{\phi}$ towards the good-state, i.e. to the e.g. the vertical axis.\\
+The rotation is implemented as for Grover by defining a product of two reflections: a reflection trough the bad state and a reflection through $\ket{U}$.
+The first reflection is realized by a circuit $S_{\phi_0}$ which can distinguish the good from the bad state by putting a minus in front of $\ket{\phi_1}$ and leaving $\ket{\phi_0}$ alone.
+In case the last qubits are $\ket{1}, \ket{0}$ for $\ket{\phi_1}, \ket{\phi_0}$ respectively, then the reflection would be
+\begin{equation}\label{eq:PhiReflection}
+  S_{\psi_0} = 1-2\ket{\psi_0}\ket{0}\bra{\psi_0}\bra{0},
+\end{equation}
+and can in turn be realized by applying a Z-gate to the last qubit.\\
+The second reflection can be implemented as $AR_{0}A^{\dagger} = AR_0A$ with $R_0$
+\begin{equation}\label{eq:ZeroReflection}
+  R_0 = 2\ket{0^n}\bra{0^n} - \mathbb{1}.
+\end{equation}
+ $AR_0A$ puts a minus sign in front of states orthorgonal to $\ket{U}$. The product of both reflection is then (thanks to Grover) a rotation of angle
+ $2\theta$ in the two-dimensional picture introduced above. \\
+ The following amplitude amplification procedure increases the amplitude of the good state close to 1:
+\begin{enumerate}
+  \item{Setup the starting state $\ket{U} = A\ket{0^n}$}.
+  \item{Repeat the following $\mathcal{O}(1 \sqrt{a})$}.
+  \begin{itemize}
+    \item Reflect through bad state $\ket{\phi_0}$ ($R_G$).
+    \item Reflect through $\ket{U}$ ($AR_0A$).
+  \end{itemize}
+\end{enumerate}
+After k iterations the state is 
+\begin{equation}\label{eq:Qktimes}
+\mathcal{Q}^k \ket{U} = \sin \biggl[ (2k+1)\theta_a \biggr] \ket{\phi_1} + \cos \biggl[ (2k+1)\theta_a \biggr] \ket{\phi_0}.
+\end{equation}
+We would like to end up with an angle $(2k+1)\theta_a \approx \pi/2$ to be close to $\sin(pi/2) \approx 1$.\\
+The above explanation describes how amplitude amplification is applied to the initial state $\ket{\phi}$ in order to amplify the good state $\ket{phi_1}$.
+But how is this actually connected to Quantum Phase Estimation which then gives a estimation $\tilde{a}$?\\
+Given again the statevector which is defined to be living in a two-dimensional subspace $H_{\phi}$: $\ket{\phi} \in H_{\phi}$ spannend by two orthogonal and normalized
+projectors $\ket{\phi_1}, \ket{\phi_0}$ onto $H_1$ and $H_0$.\\
+The action of $\mathcal{Q} := AR_0AR_g$ is given by
+\begin{align}\label{eq:actionOfQ}
+  \mathcal{Q} \ket{\phi_0} &= (2 \cos^2(\theta_a) - 1 )\ket{\phi_0} + 2 \sin(\theta_a)cos(\theta) \ket{\phi_1}\\
+  \mathcal{Q} \ket{\phi_1} &= -2 \sin(\theta_a) \cos(\theta_a) \ket{\phi_0} 
```

Comment:
The second term is missing. This should be: (2\cos^2(\theta_a)-1)\ket{\phi_1}
of which the angle will then indeed equal \cos(2\theta_a)